src/components/eva-annotation-model.js

```diff
@@ -0,0 +1,256 @@
+export const consequenceTypeDetails = {
+    'transcript_ablation': {
+        name: 'transcript_ablation',
+        acc:'SO:0001893',
+        color: '#FF0000',
```

Comment:
This mapping should probably be exposed somehow, otherwise the user doesn't know what the colors mean.  For now maybe something like showing the consequence name on hover would work, eventually some kind of key in the sidebar (and I guess also allowing filtering?) would be cool.

Reply:
I've added a simple tooltip for now. 
I looked into more complicated tooltips but they add a lot of code or dependencies. 
It's not the most pretty but it works for now.